### PR TITLE
post - ng2 ebook announcement

### PR DESCRIPTION
**Pull request by cexbrayat**
Official announce : what do you think ? (pretty much the same content you already read, slightly changed and with pictures, on @clacote 's request)

### CODE REVIEW

File: _posts/2015-04-14-angular2-ebook.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
...ybe it will end up in blog series instead of a book. Or I might settle on a beach in Indonesia, selling cocktails, and stop chasing the hype of Web Development... But the most probable trajectory is that I will write a book on Angular2, pretty much as I did when I wrote one on AngularJS 1.x : because I love to write about things I like, and that it is for me the best way to understand things deeper. And I would like to write it in English this time. And if with the help of the Ninja Squad team, it turns into a book that can help others, that’s great! It is a pleasure to see that so many of you like the 'Pay what you want' model we have on [our first ebook](https://books.ninja-squad.com). So I think it will be the same for this one, to allow everybody to buy it, and for those who can, to support us and the charity cause we chose. And maybe I'll add a pro package this time, with exercices, quizzes and a full project, so you can jump in head first, and save a lot of time. If this sounds interesting to you, you can check [this page](https://books.ninja-squad.com/angular2) and add your email : I may send you early work versions, or just ping you when the book is done, if it is some day.
```
Comment (jnizet) [inline _posts/2015-04-14-angular2-ebook.md]: I would start a new paragraph before "And I would like to write it in English", and remove the "And" at the beginning.

**Comment by clacote** [comment]
My review in #138.

:+1: 

We need to have a permalink on first ebook, and link it when we need to mention specifically the first one, as in this post. I'll add this in ninja-squad/books. I'm just currently not sure what to serve at https://books.ninja-squad.com: a soft redirection to https://books.ninja-squad.com/angular1 (yet to be implemented), until the new ebook is on sale?

**Comment by cexbrayat** [comment]
I was thinking that we could serve a page with the 2 ebooks on books.ninja-squad.com, and move the current page to a page like /angularjs, but I didn't do anything about this...

**Comment by clacote** [comment]
Done in ninja-squad/books#78 : /angular1 serves current ebook page, `/` redirects to `/angular1`.
Do you prefer `/angularjs` as URL?

**Comment by cexbrayat** [comment]
Yes, maybe...
Do you agree with the idea of an index page at `/` listing all the ebooks (instead of redirecting) ?

**Comment by clacote** [comment]
Of course an index page would be better.
I'll change Ninja-Squad/books#78 to use `/angularjs`.
You should update your blog post to reference this new URL for first ebook.
I'll update all order blog posts in another PR.

**Comment by cexbrayat** [comment]
Ok, updated!

**Pull request merged**